KarhooUISDK/Screens/QuoteList/QouteListTable/QuoteListTable+MVPC.swift

```diff
@@ -41,7 +42,26 @@ protocol QuoteListTablePresenter: AnyObject {
 
     func updateQuoteListState(_ state: QuoteListState)
 
+    func getErrorViewModel() -> QuoteListTableErrorViewModel
```

Comment:
We discussed that for all errors should be created some ErrorViewCreator to align single responsibility and open-close principles. Why finally you put it as part of QuoteListTablePresenter ?

Reply:
Presenter needs to know when the action has been made (click on error control view). We could embed callbacks inside an error case enum, but the result would be opposite → more complex and hard to follow solution.

Reply:
In mockups there is only one action button to refresh quote. If we will decide to automatically reload quote list I will opt for extracting error logic.

Reply:
"Contact Us" is an action as well